examples/platforms/simulation/platform-simulation.h

```diff
@@ -158,6 +161,25 @@ void platformRadioDeinit(void);
  */
 void platformRadioReceive(otInstance *aInstance, uint8_t *aBuf, uint16_t aBufLength);
 
+/**
+ * This function signals Tx Done to the radio (instead of echo frames).
+ *
+ * @param[in]  aInstance  A pointer to the OpenThread instance.
+ * @param[in]  pktSeq     Packet sequence number to be confirmed as sent.
+ *
+ */
+void platformRadioTxDone(otInstance *aInstance, uint8_t pktSeq);
+
+/**
+ * This function inputs the parameter used to simulate a CCA procedure.
+ *
+ * @param[in]  aInstance  A pointer to the OpenThread instance.
+ * @param[in]  channel    Channel number in use.
+ * @param[in]  value      CCA result (0 for no activity, 255 busy).
+ *
+ */
+void platformChannelActivity(otInstance *aInstance, uint8_t channel, int8_t value);
```

Comment:
```suggestion
void platformChannelActivity(otInstance *aInstance, uint8_t channel, uint8_t value);
```
255 must be uint8_t?

Reply:
To work according to the simulator implementation. This number will vary between -128/127. There is an error there.
Thanks.